lms/templates/credit/fake_credit_eligibility_status.html

```diff
@@ -0,0 +1,27 @@
+<%namespace name='static' file='/static_content.html'/>
+<html>
+<head><title>Fake Page to Update the minimum grade course requirement status</title>
+</head>
+<body>
+<p>Fake Credit Eligibility
+<div>
+% if error:
+    <div id="errors-info" style="padding-top: 5px;color: red">${ error }</div>
+% endif
+%if status:
+    <div id="success-info" style="padding-top: 5px;color: green">Status added.</div>
+% endif
+
+<form id="course-checklists" class="course-checklists" method="post" action="">
```

Comment:
Do we need these CSS ids?  It looks like these were originally defined for a page in Studio and might have gotten copied over.